app/serializers/user_serializer.rb

```diff
@@ -1,8 +1,45 @@
 # frozen_string_literal: true
 
+USER_TYPES = {
```

Comment:
Why are these outside of the serializer?

Reply:
Cause I didn't think about putting them inside. Yeah, I guess when we access them it's nicer to know where they come from.